dev-docs/publisher-api-reference.md

```diff
@@ -901,7 +902,7 @@ Remove a callback event
 ### pbjs.buildMasterVideoTagFromAdserverTag(adserverTag, options) ⇒ `String`
 
 {: .alert.alert-danger :}
-This method is deprecated as of version [0.26.0](https://github.com/prebid/Prebid.js/releases/tag/0.26.0).  To show video ads, include the `dfpVideoSupport` module in your build, and use `pbjs.adServers.dfp.buildVideoUrl`.  For more information, see [Show Video Ads with DFP]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
+This method is deprecated as of version [0.26.0](https://github.com/prebid/Prebid.js/releases/tag/0.26.0).  Please use [`pbjs.dfp.buildVideoUrl`](#module_pbjs.dfp.buildVideoUrl) instead.
```

Comment:
Same comment as above re notation.  Seems like it should be "#module_pbjs.adServers.dfp.buildVideoUrl"